osx/app_resources/plover_launcher.c

```diff
@@ -0,0 +1,41 @@
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <libgen.h>
+
+#define ARGS_LIST_SIZE sizeof(arg_list) / sizeof(char*)
+
+extern char **environ;
+int main(int argc, char *argv[]) {
+
+    // Relative path from this file.
+    char python_path[] = "/../Frameworks/pythonexecutable";
+
+    // Get this file's path.
+    char path[1048];
+    uint32_t size = sizeof(path);
+    _NSGetExecutablePath(path, &size);
+
+    // Remove filename from path.
+    char * last_slash;
+    last_slash = strrchr(path, '/');
+    path[last_slash - path] = '\0';
+
+    // Join file path and relative path to file.
+    strcat(path, python_path);
```

Comment:
It's best to avoid `strcat`: it's a common source of buffer overruns (if the buffer is not big enough).